data_structures/test/sqlist_test.c

```diff
@@ -0,0 +1,129 @@
+/**
+ * @file
+ * @brief Test for sqlist.c
+ * @details Test for insert, delete, search, locate operations of sqlist
+ * @author danielhu19 (shady030314@gmail.com)
+ */
+#include <list/sqlist.h>
+
+/**
+ * @test Test insert operations
```

Comment:
AFAIK, this needs to be added for Doxygen to detect them (same for the functions below).

```suggestion
 * @test Test insert operations
 * @returns void
```